ui-components/src/config/NavigationContext.tsx

```diff
@@ -5,7 +5,7 @@ type Url = UrlObject | string
 
 export interface LinkProps {
   href: string
-  aria?: Record<string, string>
+  ariaProps?: Record<string, string>
```

Comment:
open to other ideas for how to name this to not collide with the reserved prop of "aria"